spec/services/applications/accept_spec.rb

```diff
@@ -102,6 +102,7 @@
       it "rejects other_application" do
         service.accept
         expect(application.reload.lead_provider_approval_status).to eql("accepted")
+        expect(application.reload.accepted_at).not_to be_nil
```

Comment:
Would it be better to check for the date rather than not `nil` here? so `be_within(1.minute).of(Time.zone.now)`